sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/LogsHelper.cs

```diff
@@ -80,7 +80,7 @@ internal static string GetMessageAndSetProperties(LogRecord logRecord, IDictiona
 
             if (!string.IsNullOrEmpty(logRecord.EventId.Name))
             {
-                properties.Add("EventName", logRecord.EventId.Name);
+                properties.Add("EventName", logRecord.EventId.Name.Truncate(SchemaConstants.KVP_MaxValueLength));
             }
 
             return message;
```

Comment:
Truncate message?

Reply:
I'm wary of doing too much truncation in the `LogsHelper` class because this class is referenced by multiple types and it's tough to verify where these values are going.

In this case the `message` gets passed into `LogsHelper.ConvertExceptionTree()` which is a recursive method that can overwrite `message`.

Reply:
Makes sense